csharp/Test.cmd

```diff
@@ -5,19 +5,33 @@ SET CMDHOME=%~dp0
 @REM Remove trailing backslash \
 set CMDHOME=%CMDHOME:~0,-1%
 
-SET VSIDEDIR=%VS120COMNTOOLS%..\IDE
-SET VSTESTEXEDIR=%VSIDEDIR%\CommonExtensions\Microsoft\TestWindow
-SET VSTESTEXE=%VSTESTEXEDIR%\VSTest.console.exe
+if "%NUNITCONSOLE%" == "" (
+  GOTO :NUNITCONSOLEERROR
+)
 
 cd "%CMDHOME%"
 @cd
 
-SET CONFIGURATION=Release
+SET CONFIGURATION=Debug
```

Comment:
@kai-zeng, FYI - I deleted StackTrace-based validations this week (https://github.com/Microsoft/SparkCLR/commit/59fac3e7c230150f8587b330d98b86a4a57e16a8)